### PR TITLE
Frcnn 50 v1d

### PR DESCRIPTION
**Pull request by zhreshold**
It took a while to upload the pretrained model, for those who want to try it now, use 

```python
net = gcv.model_zoo.get_model('faster_rcnn_resnet50_v1d_voc', pretrained=False)
net.load_parameters('faster_rcnn_resnet50_v1d_voc_0017_0.8118.params')
```

Split weights due to github limitation:

https://gist.github.com/zhreshold/e915d864b3c32ae37c48d2515a86d7be/raw/8adf623c6b9cd1c84b893c6cfdc6c29dc08bea20/faster_rcnn_resnet50_v1d_voc_0017_0.8118.params.partaa
https://gist.github.com/zhreshold/e915d864b3c32ae37c48d2515a86d7be/raw/8adf623c6b9cd1c84b893c6cfdc6c29dc08bea20/faster_rcnn_resnet50_v1d_voc_0017_0.8118.params.partab
https://gist.github.com/zhreshold/e915d864b3c32ae37c48d2515a86d7be/raw/8adf623c6b9cd1c84b893c6cfdc6c29dc08bea20/faster_rcnn_resnet50_v1d_voc_0017_0.8118.params.partac

Download and `cat faster_rcnn_resnet50_v1d_voc_0017_0.8118.params.part* > faster_rcnn_resnet50_v1d_voc_0017_0.8118.params`

### CODE REVIEW

**Comment by mli** [comment]
Job PR-634-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-634/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by WalterMa** [comment]
In this PR, seems there are not included consine lr and label smooth tricks refer to issue https://github.com/dmlc/gluon-cv/issues/586?

**Comment by zhreshold** [comment]
I will update training script later

**Comment by zhreshold** [comment]
close as it's obsolete
